src/types.ts

```diff
@@ -715,3 +715,13 @@ export interface DepositTransaction {
   value: number;
   txHash: string;
 }
+
+export interface LockupInfo {
+  endAt: number;
```

Comment:
question) what does this field mean?

Reply:
It's a timestamp(ms) for lockup end date.